### ISSUE TITLE
TDLib v1.8.0 support?

### ISSUE THREAD

**Issue by s3rj1k** [Arman92/go-tdlib#102]
TDLib v1.8.0 support?

**Comment by 0cv** [comment]
Indeed, I'm getting an error similar to this post here where APIID and the HASH are not read correctly, and appear to be null 
https://github.com/zevlg/telega.el/issues/374

Not sure what shall be changed as api_id seems to be correct 

<img width="1194" alt="image" src="https://user-images.githubusercontent.com/1253866/201367319-1ca94662-35eb-4659-ab79-8d84dc5abff8.png">

EDIT: I've reinstalled the branch v1.7 of TDLib and this worked.

EDIT2: v1.8.0 branch of tdlib is **still** working. However there seems to be some breaking changes in the master branch

**Comment by isaquecsilva** [comment]
Hi! Great to know! Im facin same problem right now.

I'll give a try in an older version.
